client/pages/clients/[cid].js

```diff
@@ -0,0 +1,29 @@
+function Client({ client }) {
+    return (
+        <div>
+            <p>name: {client.data.name}</p>
+            <p>email: {client.data.email}</p>
+            <p>address: {client.data.address}</p>
+        </div>
+    )
+}
+
+export async function getServerSideProps(context) {
+    const id = context.query.cid
+    const res = await fetch(`http://localhost:8080/client/${id}`)
```

Comment:
something to fix at some point: the base api URL should be an env variable

Reply:
absolutely! not only that, but also a ton of stuff in main.go should be env variables as well.